portlets/src/main/webapp/vue-app/engagement-center/components/rules/RuleDetailDrawer.vue

```diff
@@ -287,7 +287,10 @@ export default {
           }
           this.$engagementCenterUtils.displayAlert(msg, 'error');
         })
-        .finally(() => this.$refs.ruleDetailDrawer.endLoading());
+        .finally(() => {
+          this.$refs.ruleDetailDrawer.endLoading();
+          this.canAnnounce = false;
+        });
```

Comment:
same as previous remark in fact

```suggestion
        .finally(() => this.sending = false);
```

Reply:
done